mkt/extensions/models.py

```diff
@@ -36,10 +36,11 @@
 
 class ExtensionManager(ManagerBase):
     def pending(self):
-        return self.filter(versions__status=STATUS_PENDING).order_by('id')
+        return self.filter(
+            disabled=False, versions__status=STATUS_PENDING).order_by('id')
 
     def public(self):
-        return self.filter(status=STATUS_PUBLIC).order_by('id')
+        return self.filter(disabled=False, status=STATUS_PUBLIC).order_by('id')
 
```

Comment:
I generally write a helper if it'd be used more than once (i.e. here + pending), but it's a very minor point.